resources.whatwg.org/build/deploy.sh

```diff
@@ -25,6 +25,7 @@ NEW_SERVER_PUBLIC_KEY="ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIb
 TRAVIS=${TRAVIS:-false}
 TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST:-false}
 DEPLOY_USER=${DEPLOY_USER:-}
+EXTRA_FILES=${EXTRA_FILES:-}
```

Comment:
Added this after local testing revealed an error: "EXTRA_FILES: unbound variable"

With that fix, it works locally, so merging this.